src/Compilers/VisualBasic/Portable/Symbols/EventSignatureComparer.vb

```diff
@@ -26,15 +26,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         Public Shared ReadOnly ExplicitEventImplementationComparer As EventSignatureComparer =
             New EventSignatureComparer(considerName:=False,
```

Comment:
Makes sense, but not essential for this PR or RC. Filed issue [https://github.com/dotnet/roslyn/issues/14408](https://github.com/dotnet/roslyn/issues/14408)

---
In reply to: [82654149](https://github.com/dotnet/roslyn/pull/14229#discussion_r82654149) [](ancestors = 82654149)